File: C#/Shooter.cs
using UnityEngine;
using System.Collections;

public class Shooter : MonoBehaviour {

	// Variables
	
	public Rigidbody bullet;
	public float power = 1500f;
	public float movespeed = 2f;
	
	// Update is called once per frame
	void Update () {
		
		// Control the movement of the camera|position of the projectile
		float h =Input.GetAxis("Horizontal") * Time.deltaTime * movespeed;
		float v =Input.GetAxis("Vertical") * Time.deltaTime * movespeed;
		
		transform.Translate(h,v,0);
		
		if (Input.GetButtonUp("Fire1")) 
		{
			//Create an instance of the bullet
			Rigidbody instance = Instantiate(bullet,transform.position,transform.rotation) as Rigidbody; 
			
			//Add force to the instantiated object
			Vector3 fwd = transform.TransformDirection(Vector3.forward);
			
			instance.AddForce(fwd * power);
		}
	
	}
}

File: Javascript/Shooter.js
#pragma strict

var bullet: Rigidbody;
var power : float = 1500;
var movespeed : float = 2;

function Update () {

	var h : float = Input.GetAxis("Horizontal") * Time.deltaTime * movespeed;
	var v : float = Input.GetAxis("Vertical") * Time.deltaTime * movespeed;
	
	transform.Translate(h,v,0);
	
	if (Input.GetButtonUp("Fire1")) 
	{
		//Create an instance of the bullet
		var instance : Rigidbody = Instantiate(bullet,transform.position,transform.rotation);
		
		// Add force to the instantiated object
		var fwd : Vector3 = transform.TransformDirection(Vector3.forward);
		
		instance.AddForce(fwd * power);
	}
}